src/scales/scale.time.js

```diff
@@ -39,7 +38,7 @@ module.exports = function(Chart) {
 		}
 	};
 
-	var TimeScale = Chart.Scale.extend({
+	var TimeScale = Chart.TimeScaleBase.extend({
 		initialize: function() {
```

Comment:
I think this `initialize` override needs to be removed (looks exactly the same as the one in `TimeScaleBase`).